code/modules/research/designs/tool_designs.dm

```diff
@@ -63,6 +63,16 @@
 	category = list("Tool Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING
 
+/datum/design/rld
```

Comment:
This should be in a separate file in the NSV13 folder.

Reply:
ok so i should add 2 files in the nsv13 folder 1 for tool designs and 1 for tech nodes?

Reply:
have done that now